client/src/lib/catchFormArrays.js

```diff
@@ -0,0 +1,137 @@
+const fishtypes = [
+  "Aal",
+  "Aesche",
+  "Atlantischer Lachs",
+  "Bachforelle",
+  "Bachsaibling",
+  "Brassen/Brachse/Blei",
+  "Dorsch",
+  "Europäischer Wels/Waller",
+  "Flussbarsch",
+  "Hecht",
+  "Hering",
+  "Hornhecht",
+  "Karpen",
+  "Meerforelle",
+  "Rapfen",
+  "Regenbogenforelle",
+  "Schleie",
+  "Stint",
+  "Wolfsbarsch",
+  "Zander",
+  "Rotfeder",
+  "Rotauge(Plötze)",
+  "Güster",
+  "Karausche",
+  "A n d e r e",
+];
+
+const dates = {
+  days: [
+    "01",
+    "02",
+    "03",
+    "04",
+    "05",
+    "06",
+    "07",
+    "08",
+    "09",
+    "10",
+    "11",
+    "12",
+    "13",
+    "14",
+    "15",
+    "16",
+    "17",
+    "18",
+    "19",
+    "20",
+    "21",
+    "22",
+    "23",
+    "24",
+    "25",
+    "26",
+    "27",
+    "28",
+    "29",
+    "30",
+    "31",
+  ],
+  months: [
+    "Januar",
+    "Februar",
+    "März",
+    "April",
+    "Mai",
+    "Juni",
+    "Juli",
+    "August",
+    "September",
+    "Oktober",
+    "November",
+    "Dezember",
+  ],
+  years: ["2021", "2022"],
+  times: [
+    "00:00",
+    "00:30",
+    "01:00",
+    "01:30",
+    "02:00",
+    "02:30",
+    "03:00",
+    "03:30",
+    "04:00",
+    "04:30",
+    "05:00",
+    "05:30",
+    "06:00",
+    "06:30",
+    "07:00",
+    "07:30",
+    "08:00",
+    "08:30",
+    "09:00",
+    "09:30",
+    "10:00",
+    "10:30",
+    "11:00",
+    "11:30",
+    "12:00",
+    "12:30",
+    "13:00",
+    "13:30",
+    "14:00",
+    "14:30",
+    "15:00",
+    "15:30",
+    "16:00",
+    "16:30",
+    "17:00",
+    "17:30",
+    "18:00",
+    "18:30",
+    "19:00",
+    "19:30",
+    "20:00",
+    "20:30",
+    "21:00",
+    "21:30",
+    "22:00",
+    "22:30",
+    "23:00",
+  ],
+};
+
+const tackle = [
+  "A n d e r e",
+  "Grundangeln",
+  "Spinnfischen",
+  "Fliegenfischen",
+  "Posenangeln",
+];
```

Comment:
Thomas would like to see "Dynamitfischen" as well ;)
```suggestion
const tackle = [
  "A n d e r e",
  "Grundangeln",
  "Spinnfischen",
  "Fliegenfischen",
  "Posenangeln",
  "Dynamitfischen"
];
```

Reply:
Sorry, but nope. ;)